src/API/Google/AdsAssetGroup.php

```diff
@@ -82,17 +86,56 @@ public function __construct( GoogleAdsClient $client, AdsAssetGroupAsset $asset_
 		$this->asset_group_asset = $asset_group_asset;
 	}
 
+	/**
+	 * Create an asset group.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param int $campaign_id
+	 *
+	 * @return id The asset group id.
+	 * @throws ExceptionWithResponseData ExceptionWithResponseData When an ApiException is caught.
+	 */
+	public function create_asset_group( int $campaign_id ): int {
+		try {
+			$campaign_resource_name = ResourceNames::forCampaign( $this->options->get_ads_id(), $campaign_id );
+			$current_date_time      = ( new DateTime( 'now', wp_timezone() ) )->format( 'Y-m-d H:i:s' );
+			$name                   = sprintf(
```

Comment:
💅  I'd personally would change `$name` to `$asset_group_name` since $name is quite generic. (Especially for the function signature)